tests/test_connect.py

```diff
@@ -89,14 +95,15 @@ def test_disconnect_on_no_connection(self):
     def test_ssl_tunnel(self):
         connect.SoapStubAdapter('sdkTunnel', 8089, httpProxyHost='vcsa').GetConnection()
 
-    @tests.VCRTestBase.my_vcr.use_cassette('ssl_tunnel_http_failure.yaml',
-                      cassette_library_dir=tests.fixtures_path,
-                      record_mode='none')
-    def test_ssl_tunnel_http_failure(self):
-        from six.moves import http_client
-        def should_fail():
-            connect.SoapStubAdapter('vcsa', 80, httpProxyHost='vcsa').GetConnection()
-        self.assertRaises(http_client.HTTPException, should_fail)
+    @patch('six.moves.http_client.HTTPSConnection')
+    def test_ssl_tunnelwith_cert_file(self, hs):
```

Comment:
@pgbidkar I added this test to validate the parameter passing of `cert_file` and `key_file`.